src/pybind/mgr/dashboard/frontend/src/app/shared/components/back-button/back-button.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Location } from '@angular/common';
+import { Component, Input } from '@angular/core';
+import { I18n } from '@ngx-translate/i18n-polyfill';
+
+@Component({
+  selector: 'cd-back-button',
+  templateUrl: './back-button.component.html',
+  styleUrls: ['./back-button.component.scss']
+})
+export class BackButtonComponent {
+  constructor(private location: Location, private i18n: I18n) {}
+
+  @Input() name: string = this.i18n('Back');
+  @Input() back: Function = () => this.location.back();
```

Comment:
Would it be possible to determine the parent of the current page and go back to that?
Otherwise we can get different results from what we had before.
p.e. If you open a new tab, then open an "RBD Add"  page and click `Back`, it will go back to the start page of chrome.

Reply:
This is the right behavior. As the button doesn't say go back to RBD listing. I assume that this is also a edge case, most likely the user will go via button to the form. Also this behavior is needed if you go to the silence form from the alert page - now you go back to the alert page if you came from it.